libavcodec/vaapi_encode.c

```diff
@@ -2012,6 +2047,8 @@ static av_cold int vaapi_encode_init_slice_structure(AVCodecContext *avctx)
         return 0;
     }
 
+    av_assert0(ctx->slice_block_height > 0 && ctx->slice_block_width > 0);
```

Comment:
Sure, will fix this.

Reply:
This is a guard to avoid slice_block_height&slice_block_width equals to 0 as the denominator. Put it here can also remind develop set correct slice_block_height&slice_block_width in ctx->codec->block_size when it defined. So it is better to keep this here.